content/pl/docs/tutorials/kubernetes-basics/explore/explore-intro.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Pody i Węzły
+weight: 10
+---
+
+## {{% heading "objectives" %}}
+
+* Poznać Pody Kubernetesa.
+* Poznać węzły Kubernetes.
+* Nauczyć się rozwiązywać problemy z aplikacjami.
+
+## Pody Kubernetes {#kubernetes-pods}
+
+{{% alert %}}
+_Pod to grupa jednego lub wielu kontenerów aplikacji (jak np. Docker) zawierających współdzieloną przestrzeń dyskową (volumes), adres IP i informacje, jak mają być uruchamiane._
```

Comment:
I belive this could be improved to something around that:

> Pod to grupa jednego lub kilku kontenerów współdzielących przestrzeń dyskową i sieciową, wraz z konfiguracją określającą sposób ich uruchamiania.